pom.xml

```diff
@@ -459,7 +459,7 @@
             <dependency>
                 <groupId>io.airlift.tpch</groupId>
                 <artifactId>tpch</artifactId>
-                <version>0.5</version>
+                <version>0.6</version>
```

Comment:
Because newer Tempto version is dependent on Tpch 0.6. This migration commit is already on master. I think you shouldn't review it. The very next sprint branch will aready have this commit from master.